.github/workflows/ci.yml

```
@@ -128,21 +183,71 @@
+              const jsonStart = content.indexOf('{');
+              if (jsonStart >= 0) {
+                audit = JSON.parse(content.substring(jsonStart));
+              } else {
+                console.log('No JSON output from audit, assuming no vulnerabilities');
+                process.exit(0);
+              }
+            } catch (e) {
+              console.log('Could not parse audit output, assuming no vulnerabilities');
+              process.exit(0);
+            }
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)